.golangci.yml

```diff
@@ -88,6 +100,6 @@ issues:
 # golangci.com configuration
 # https://github.com/golangci/golangci/wiki/Configuration
 service:
-  golangci-lint-version: 1.52.x # use the fixed version to not introduce new linters unexpectedly
+  golangci-lint-version: 1.62.1 # use the fixed version to not introduce new linters unexpectedly
   prepare:
```

Comment:
This whole section is deprecated - I would just remove it.

Reply:
Now removed.